src/modules.py

```diff
@@ -5,27 +5,30 @@
 level operating system functionality.
 """
 
+import os
 import sys
 # See docs for the sys module: https://docs.python.org/3.7/library/sys.html
 
 # Print out the command line arguments in sys.argv, one per line:
-# YOUR CODE HERE
+print(sys.argv[0])
+print(sys.argv[1])
+print(sys.argv[2])
+
 
 # Print out the OS platform you're using:
-# YOUR CODE HERE
+print(os.uname())
 
 # Print out the version of Python you're using:
-# YOUR CODE HERE
+print(sys.version_info)
 
 
-import os
 # See the docs for the OS module: https://docs.python.org/3.7/library/os.html
 
 # Print the current process ID
-# YOUR CODE HERE
+print(os.getpid())
```

Comment:
You'll be excited to look back at this section once you learn more about processes in C and realize how much Python lets you work with the OS.